src/components/accordion-disclosure/types.ts

```diff
@@ -28,4 +31,26 @@ export interface AccordionDisclosureProps {
 	 * descriptive.
 	 */
 	title: ReactNode
+
+	/**
+	 * Whether this disclosure is the first in a group of similar items.
+	 * If `true`, specific styles for the first item in a group will be applied.
+	 * If `false`, then styles will be applied assuming there is a previous item.
+	 * If this prop & `isLastItem` are `undefined`, no group styles are used.
+	 */
+	isFirstItem?: boolean
+
+	/**
+	 * Whether this disclosure is the last in a group of similar items.
+	 * If `true`, specific styles for the last item in a group will be applied.
+	 * If `false`, then styles will be applied assuming there is a next item.
+	 * If this prop & `isFirstItem` are `undefined`, no group styles are used.
+	 */
+	isLastItem?: boolean
```

Comment:
💭 What if we put these in a `group` prop? 

```
group?: {
  numItems: number
  currentIndex: number
}
```
Or we could pass the `isFirst` `isLast` in the group prop instead of the num / index. Then we are explicitly stating that the item is in a group as a prop instead of inferring whether its in a group from `isFirst` or `isLast`.

Reply:
I dig this! 👍 will update...

Reply:
✅ updated! I went with `groupData` but otherwise kept things exactly as suggested. Definitely feels cleaner, and dig having the `isFirstItem` & `isLastItem` logic within the component 👍